public/app/app.ts

```diff
@@ -156,8 +159,12 @@ export class GrafanaApp {
       const modalManager = new ModalManager();
       modalManager.init();
 
-      // Preload selected app plugins
-      await preloadPlugins(config.pluginsToPreload);
+      await Promise.all([
+        loadLocalePromise,
+
+        // Preload selected app plugins
+        await preloadPlugins(config.pluginsToPreload),
```

Comment:
do we need to use await for this promise?

Reply:
no! good catch!